src/main/java/com/redhat/devtools/lsp4ij/settings/ui/LanguageServerPanel.java

```diff
@@ -180,7 +180,7 @@ private static FormBuilder addTab(JBTabbedPane tabbedPane, String tabTitle, bool
         if (addToTop) {
             tabPanel.addToTop(builder.getPanel());
         } else {
-            tabPanel.add(builder.getPanel());
+            tabPanel.addToCenter(builder.getPanel());

```

Comment:
This is actually behaviorally the same, but it made sense to me to be explicit since the panel is definitely for a `BorderLayout`.